LICENSE

```diff
@@ -0,0 +1,13 @@
+© 2014–2018 UC Regents, Creative Commons Attribution 4.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
FWIW - this text may not be detected by github as Apache 2.0 since it doesn't match their version of the Apache license text see https://help.github.com/articles/adding-a-license-to-a-repository/

I'm not sure how important that is in this instance.  If you wanted Github to correctly detect the license version you probably want to give the full apache text that Github generates and just add the copyright line at the beginning.